README.md

```diff
@@ -79,17 +87,39 @@ See [the wiki](https://github.com/Parsely/wp-parsely/wiki/Setting-up-a-WP-plugin
 
 ## Screenshots
 
-1. The main settings screen of the wp-parsely plugin  
+1. The main settings screen of the WP-Parsely plugin  
```

Comment:
The plugin isn't called `WP-Parsely`. It's called `Parse.ly`, and has a slug of `wp-parsely`. Let's aim not to mix the two.
```suggestion
1. The main settings screen of the Parse.ly plugin.  
```